kt/godot-library/src/main/kotlin/godot/core/Variant.kt

```diff
@@ -642,6 +642,18 @@ enum class VariantType(
         }
     ),
 
+    JVM_ULONG(
```

Comment:
I'm not for overloading Variant with even more fake types that are not part of the actual C++ Variant. We introduced JVM_INT in the first place so people can register methods with Int parameters instead of only Long. But IMO it was a poor way to implement it so I don't wish to make it worse.
 It's not like we are supporting registering unsigned integers in the first place so it's not necessary to provide a 2 way conversion.
 The casting shouldn't directly be in the implementation of the api methods when necessary.

Reply:
Exactly, JVM_INT is done the same way, adding cast instead would just break api gen flow.
If the reason for JVM variant being "poor" implementation is only "this does not exists in cpp" I think you should reconsider it. This implementation makes entry gen and api gen code much easier